### ISSUE TITLE
Android - Workspace upgrade - Offline indicator is below the content

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#44971]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___
Found when validating https://github.com/Expensify/App/pull/43822
**Version Number:** 9.0.5-3
**Reproducible in staging?:** y
**Reproducible in production?:** no, this is the new feature
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause internal team
**Slack conversation:** 

## Action Performed:
1. Launch New Expensify app.
2. Go to Workspace settings.
3.  Paste the following link in any chat and tap on it (https://staging.new.expensify.com/settings/workspaces/31DF97D37125FBD6/upgrade/report-fields)
4. Go offline.
## Expected Result:
The offline indicator will be at the bottom of the screen.

## Actual Result:
The offline indicator is below the content.

## Workaround:
unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<img src="https://github.com/Expensify/App/assets/38435837/0bc204bc-86ea-4a33-b8af-9523fb476f11" width="40%" height="40%">

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by m-natarajan** [comment]
@joekaufmanexpensify FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by garrettmknight** [comment]
I don't think this needs to be a blocker. I'm going to take this over as part of the upgrades project.

**Comment by joekaufmanexpensify** [comment]
Sounds good, yeah agree on this not being a blocker. TY for grabbing!

**Comment by allroundexperts** [comment]
@cretadn22's [proposal](https://github.com/Expensify/App/issues/44971#issuecomment-2214220680) looks good to me. It's a pretty straight forward fix.

🎀 👀 🎀 C+ reviewed

**Comment by hayata-suenaga** [comment]
sorry for the delay, assigning @cretadn22 😄

**Comment by garrettmknight** [comment]
@cretadn22 when do you think you'll have the PR up?

**Comment by garrettmknight** [comment]
Hit staging today, should be on prod tomorrow and trigger the automation.

**Comment by garrettmknight** [comment]
**Payment summary:**
- Contributor: @allroundexperts $250
- Reviewer: @ishpaul777 $250

@ishpaul777 offer out to you here: https://www.upwork.com/ab/applicants/1813979251771429535/hires?filter=ish&page=1

**Comment by garrettmknight** [comment]
Oops, posted to the wrong issue!

**Comment by garrettmknight** [comment]
@cretadn22 offer out to you here: https://www.upwork.com/nx/wm/offer/103252666

**Comment by garrettmknight** [comment]
**Payment Summary:**
- Contributor: @cretadn22 $250 via Upwork
- Reviewer: @allroundexperts will handle payment for the whole project

**Comment by garrettmknight** [comment]
All paid up, closing!

**Comment by allroundexperts** [comment]
Checklist not needed here as this was a small follow up to the upgrade flow.
